pages/CreatingAnExploration.html

```diff
@@ -12,13 +12,13 @@ <h2>
     do so, click the "Create" button in the top navbar.
   </p>
 
-  <img src="static/images/libraryCreateButton.png" style="width: 100%;">
+  <img src="static/images/libraryCreateButton.png" style="width: 100%;" alt="A top-navigation bar having various options such as 'library', 'about' and 'donate' along with a 'create' button circled in yellow. ">
 
   <p>
     This will open the Exploration Editor page.
   </p>
 
-  <img src="static/images/createNewExploration.png" style="width: 100%;">
+  <img src="static/images/createNewExploration.png" style="width: 100%;" alt="Navigation bar reads 'Untitled Exploration'. A which card having some introductory content is displayed.">
```

Comment:
Second sentence does not parse, please proofread. (You can check what the code reviewer is seeing here: https://github.com/oppia/oppia.github.io/pull/47/files)